packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -475,7 +475,7 @@ const getDeltaVertex = (
 const getChange = (
     coords: readonly vec.Vector2[],
     delta: vec.Vector2,
-    constraintOpts: Omit<ConstraintArgs, "point">,
+    constraintOpts: {snapStep: vec.Vector2; range: [Interval, Interval]},
```

Comment:
Question: Do we lose anything by disconnecting `constraintOpts` from `ConstraintArgs`? I think I like it being more clearly stated like this, but curious if there was a reason to have it there other than to not repeat the types.

Reply:
We don't lose any type safety by doing this. It is more verbose to write the type out explicitly, but I agree that it's clearer.